src/index.js

```diff
@@ -1,10 +1,10 @@
 import { version } from '../package.json';
-import { toM3u8 } from './toM3u8';
+import { toM3u8, generateSidxKey } from './toM3u8';
```

Comment:
we duplicate this exact function in vhs right now. Seems better to have one source of truth.